src/migration/1727578711381-VordrUser.ts

```diff
@@ -0,0 +1,91 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class VordrUser1727578711381 implements MigrationInterface {
+  name = 'VordrUser1727578711381'
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`CREATE INDEX "IDX_post_flags_vordr" ON post (((flags->'vordr')::boolean))`);
+
+    await queryRunner.query(`CREATE OR REPLACE FUNCTION vordr_update_flags()
+RETURNS TRIGGER AS $$
+BEGIN
+  IF NEW.flags @> '{"vordr": true}' THEN
+    -- If vordr on user is set to true
+    -- Set banned and vordr to true in post table, and set vordr to true in comment table
+    UPDATE
+      post
+    SET
+      "banned" = TRUE,
+      "flags" = jsonb_set(jsonb_set(flags,
+        '{vordr}', 'true', true),         -- set vordr to true
+        '{banned}', 'true', true)         -- set banned to true
+    WHERE
+      ("authorId" = NEW.id OR "scoutId" = NEW.id)
+      AND "banned" = FALSE;
```

Comment:
@idoshamun I added a check here to not set vordr on already banned posts. Since it would basically negate the check when un-vordrering, since banned posts would then match the `(flags->'vordr')::boolean = true` statement.

Reply:
The only issue I can think of now is that banning the posts will result in penalization lost which will make this process very visible. wdyt?

Reply:
Penalization of the old posts by them?

Reply:
yes. every time we ban a post it penalized the author/scout

Reply:
But this is when we vordr an entire account?

Reply:
yes, instead of banning let's just set `showOnFeed` false as well as `vordr`

Reply:
👍 will change

Perhaps we also could adjust the API so that vordr flag has the same effect as showOnFeed, so that we don't need to set that too?

Reply:
We will need to change the feed as well. It's just too many places I think

Reply:
@idoshamun changed to be `showOnFeed=false` instead of `banned=true`